src/lib/db.test.ts

```diff
@@ -56,4 +58,22 @@ describe("db", () => {
       .then(() => db.disconnect());
   });
 
+  it("can parse document text content", () => {
```

Comment:
Added a simple deserialization test...probably worth thinking more about general Firebase testing strategy. Since we'll be changing the saving model, I didn't go crazy on tests.